### PR TITLE
Updated automatic clone&build scripts

### PR DESCRIPTION
**Pull request by babrodtk**
Basically a beautification of the vagrant virtual machine setup
This PR has scripts that build ert, several (not all) OPM modules, and ResInsight automatically

### CODE REVIEW

File: vagrant/Vagrantfile [hunk 1]
```diff
@@ -58,7 +58,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # View the documentation for the provider you're using for more
   # information on available options.
 config.vm.provider "virtualbox" do |v|
-  v.memory = 2048
+  #v.gui = true
+  v.memory = 8096
```
Comment (atgeirr) [inline vagrant/Vagrantfile]: Did you really mean 4096? Or 8192?

**Comment by atgeirr** [comment]
Very nice! Will merge (but indicate if you wanted 4 or 8 gigs of RAM for the VM, also: was it necessary because of compiler memory usage?).

File: vagrant/Vagrantfile (hunk 1 above)
Comment (babrodtk) [inline vagrant/Vagrantfile]: Nice catch;) I did no thorough testing, but 2048 was too little at least. When trying anything else with that amount of memory whilst compiling, the OS would return an out-of-memory error. I believe 8192 would be the best choice, and hopefully not too demanding for todays laptops/desktops.

**Comment by babrodtk** [comment]
Updated to a true power of two, it seems my look-up table for power-of-twos was somehow corrupt.

**Pull request merged**